radixconfig.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: radix.equinor.com/v1
+kind: RadixApplication
+metadata:
+  name: data-marketplace
+spec:
+  environments:
+  - name: prod
+  components:
+  - name: data-marketplace-prod
+    publicPort: http
+    ports:
+    - name: http
+      port: 3000
+    environmentConfig:
+    - environment: prod
+      resources:
+        requests:
+          memory: 512Mi
+          cpu: 100m
+        limits:
+          memory: 1Gi
+          cpu: 1000m
+      variables:
```

Comment:
I guess this is correct, but it just looks funny with two variables following each other

Reply:
Yeah, could omit them, but we will soon be using them